modules/web/src/app/wizard/step/cluster/component.ts

```diff
@@ -211,6 +218,13 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
       [Controls.IPv4CIDRMaskSize]: this._builder.control(null),
       [Controls.IPv6CIDRMaskSize]: this._builder.control(null),
       [Controls.NodeLocalDNSCache]: this._builder.control(false),
+      [Controls.ExposeStrategy]: this._builder.control(null),
+      [Controls.IPv4APIServerAllowedIPRanges]: this._builder.control(this._defaultAllowedIPRange, [
+        IPV4_CIDR_PATTERN_VALIDATOR,
+      ]),
+      [Controls.IPv6APIServerAllowedIPRanges]: this._builder.control(this._defaultAllowedIPv6Range, [
+        IPV6_CIDR_PATTERN_VALIDATOR,
+      ]),
```

Comment:
Do we need `_dualStackRequiredIfValidator` here as well similar to Services CIDR and Allowed IP Range controls? Or is one empty value allowed in `apiServerAllowedIPRanges.cidrBlocks` array?

Reply:
One empty value should be allowed in this case.